### PR TITLE
Update IE data for Node[List] API

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR updates the Internet Explorer data for the Node and NodeList APIs based upon results from the mdn-bcd-collector project.  Results were manually verified for confirmation.

### CODE REVIEW

File: api/Node.json [hunk 1]
```diff
@@ -171,7 +171,7 @@
               "version_added": null
             },
             "ie": {
-              "version_added": null
+              "version_added": false
```
Comment (foolip) [inline api/Node.json:174]: This `baseURIObject` entry should definitely be removed. Want to do that in a separate PR?

File: api/NodeList.json [hunk 2]
```diff
@@ -163,7 +163,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "8"
```
Comment (foolip) [inline api/NodeList.json:166]: `NodeList` is what's used for `childNodes`, but the `api.Node.childNodes` entry says IE5 and I've confirmed it's in IE7 at least. So what kind of object did IE return for `childNodes` before? I suspect this is just a problem of interface object not being exposed, and this should be set to an earlier version.

Comment (vinyldarkscratch) [reply api/NodeList.json]: I've added a custom test for this, and have updated the results manually!

**Review by foolip — APPROVED** [review]
